tests/Bridge/Symfony/Routing/ApiLoaderTest.php

```diff
@@ -113,6 +134,24 @@ private function getApiLoaderWithResourceMetadata(ResourceMetadata $resourceMeta
 
         $kernelProphecy = $this->prophesize(KernelInterface::class);
         $kernelProphecy->locateResource(Argument::any())->willReturn($routingConfig);
+        $possibleArguments = [
+            'api_platform.action.get_collection',
+            'api_platform.action.post_collection',
+            'api_platform.action.get_item',
+            'api_platform.action.put_item',
+            'api_platform.action.delete_item',
+        ];
+
+        $containerInterfaceProphecy = $this->prophesize(ContainerInterface::class);
+        $containerInterfaceProphecy->reveal();
```

Comment:
I'm not pleased with it either but there is no workaround, to avoid that and benefit from the typehint there is phpSpec for that. But I don't like forcing the users to use phpSpec and you loose your code coverage as well...

Anyway let's blame @dunglas tastes in the meantime 😆